Automattic/wp-calypso
# Help: New styles for Helpful Articles and search results

#### Changes proposed in this Pull Request

* Re-style Helpful Articles and search results as per mockups
* Add images for the Helpful Articles instead of icons
* Hide everything but the contact information and search results when searching so the page doesn't get so long

**Before**

<img width="1080" alt="Screen Shot 2021-01-12 at 3 12 33 PM" src="https://user-images.githubusercontent.com/2124984/104367372-b625ab80-54e8-11eb-85fc-a677d3462f67.png">
<img width="1088" alt="Screen Shot 2021-01-12 at 3 12 49 PM" src="https://user-images.githubusercontent.com/2124984/104367376-b756d880-54e8-11eb-983c-ab8ddf2c47c3.png">
<img width="1092" alt="Screen Shot 2021-01-12 at 3 13 06 PM" src="https://user-images.githubusercontent.com/2124984/104367380-b8880580-54e8-11eb-8285-8688bc194f6f.png">

**After**

<img width="1098" alt="Screen Shot 2021-01-12 at 3 10 02 PM" src="https://user-images.githubusercontent.com/2124984/104367053-41527180-54e8-11eb-89a9-8fb89c0d53cb.png">
<img width="1098" alt="Screen Shot 2021-01-12 at 3 11 40 PM" src="https://user-images.githubusercontent.com/2124984/104367406-c2116d80-54e8-11eb-91e0-065a2a8fe8f8.png">
<img width="1085" alt="Screen Shot 2021-01-12 at 3 11 57 PM" src="https://user-images.githubusercontent.com/2124984/104367407-c2aa0400-54e8-11eb-874a-2f57787f6178.png">

#### Testing instructions

* Switch to this PR and navigate to `/help`
* Try a search; search results should appear, the rest of the content should not be visible except for the Contact us block at the bottom.
* Clearing the search field (ESC, hitting the X, or deleting the query) or reloading the page should return the page to its original state.
* Check no results found state
* Make sure there are no other visual bugs or errors. :) 

See #40870
